Formula/c/cherrybomb.rb

```diff
@@ -0,0 +1,26 @@
+class Cherrybomb < Formula
+  desc "Tool designed to valide your spec"
+  homepage "https://blstsecurity.com"
+  url "https://github.com/blst-security/cherrybomb/archive/refs/tags/v1.0.1.tar.gz"
+  sha256 "1cbea9046f2a6fb7264d82e1695661e93a759d1d536c6d1e742032e4689efe9f"
+  license "Apache-2.0"
+  head "https://github.com/blst-security/cherrybomb.git", branch: "main"
+
+  depends_on "rust" => :build
+
+  def install
+    system "cargo", "install", *std_cargo_args
+  end
+
+  test do
+    file_url = "https://raw.githubusercontent.com/blst-security/cherrybomb/main/images/api-with-examples.yaml"
```

Comment:
if this is part of source tarball, you can just do `pkgshare.install xxx` and it would be avail for the test block